rust/template/src/api.rs

```diff
@@ -168,6 +168,12 @@ impl HDDlog {
         self.prog.lock().unwrap().enable_cpu_profiling(enable);
     }
 
+    pub fn enable_timely_profiling(&self, enable: bool) {
```

Comment:
you also need to declare `ddlog_enable_timely_profiling()`, by analogy with `ddlog_enable_cpu_profiling()` in this function.